internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,65 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+// populateVolOptions update the fields in rbdVolume struct based on the request it received.
+func populateRbdVol(
```

Comment:
`populateVolOptions` to `populateRbdVol`? function name and comment is not matching

Reply:
Corrected. ptal .. thanks @Madhu-1